README.rst

```diff
@@ -1,25 +1,31 @@
-=======
-Socorro
-=======
+=========================================
+Socorro: Mozilla crash ingestion pipeline
+=========================================
 
 Socorro is a Mozilla-centric ingestion pipeline and analysis tools for
 crash reports using the `Breakpad libraries
 <http://code.google.com/p/google-breakpad/>`_.
 
+* Free software: Mozilla Public License version 2.0
+* Community Participation Guidelines `Guidelines <https://github.com/mozilla-services/socorro/blob/main/CODE_OF_CONDUCT.md>`_
+* Chat: `#crashreporting matrix channel <https://chat.mozilla.org/#/room/#crashreporting:mozilla.org>`__
+* Socorro (processor/webapp/cron jobs)
 
-Support
-=======
+  * Code: https://github.com/mozilla-services/socorro/
+  * Documentation: https://socorro.readthedocs.io/
+  * Bugs: `Report a bug <https://bugzilla.mozilla.org/enter_bug.cgi?format=__standard__&product=Socorro&component=General>`_
 
-Note: This is a Mozilla-specific product. We do not currently have the capacity
-to support external users.
+* Antenna (collector)
 
-If you are looking to use Socorro for your product, maybe you want to consider
-this non-exhaustive list of alternatives:
+  * Code: https://github.com/mozilla-services/antenna/
+  * Documentation: https://antenna.readthedocs.io/
+  * Bugs: `Report an Antenna bug <https://bugzilla.mozilla.org/enter_bug.cgi?format=__standard__&product=Socorro&component=Antenna>`_
 
-* run your own: `electron/mini-breakpad-server
-  <https://github.com/electron/mini-breakpad-server>`_
-* run your own: `wk8/sentry_breakpad <https://github.com/wk8/sentry_breakpad>`_
-* hosted/on-premise: `Backtrace <https://backtrace.io/>`_, `BugSplat <https://bugsplat.com/>`_
+
+.. Note::
+
+   This is a very Mozilla-specific product. We do not currently have the
+   capacity to support non-Mozilla uses.
```

Comment:
I merged the `README.rst` text with the text from `docs/index.rst` such that we could use the README at the top of `docs/index.rst`. That reduces some duplicate content and makes things easier to maintain.